Setup.bat

```diff
@@ -181,3 +197,11 @@ exit /b 0
 :MarkEndOfBlock
 echo Finished: %~1
 exit /b 0
+
+:ExecuteAndCheck
+%*
+if ERRORLEVEL 1 (

```

Comment:
Do we care about other error levels? We could maybe hit 2 or 3 (file and path not found respectively)

Reply:
I had the exact same thought as you and then came across this :) [link](https://stackoverflow.com/questions/39435598/why-does-the-command-if-errorlevel-1-do-result-in-an-exit-of-batch-file-proc#:~:text=if%20errorlevel%201%20echo%20Previous,an%20error%20code%20greater%200.)

Reply:
Ok fair enough, still have a preference for using `IF %ERRORLEVEL% NEQ 0` for clarities sake

Reply:
Specific to ErrorLevel it seems, insanity: https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/if

Reply:
`IF %ERRORLEVEL% NEQ 0` sounds good to me, will include that change